lib/main.dart

```diff
@@ -15,31 +22,49 @@ class MyApp extends StatelessWidget {
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      debugShowCheckedModeBanner: false,
-      title: 'Flutter Demo',
-      theme: ThemeData(
+    return FutureBuilder(
```

Comment:
Hi @daicats2019 , Please follow this official docs: [FlutterFire Authentication Docs](https://firebase.google.com/docs/auth/flutter/start)